src/library.cpp

```diff
@@ -203,7 +213,7 @@ std::string Library::getBestTargetBookId(const Bookmark& bookmark, MigrationMode
         // No bookName nor bookTitle, no way to find target book.
         return "";
     }
-    book_filter.query("title:\""+bookmark.getBookTitle() + "\"");
+    book_filter.query("title:\"" + remove_quote(bookmark.getBookTitle()) + "\"");
```

Comment:
Isn't it safer to replace the quotes with whitespace? The answer depends on how xapian indexes a string like `ABC"XYZ`.